symphony-bdk-extensions/symphony-group-extension/src/main/resources/symphony-common-definitions.yaml

```diff
@@ -0,0 +1,308 @@
+openapi: 3.0.0
```

Comment:
Is this swagger spec file stored somewhere else? I would expect so and that we download directly form this repo instead of copying it here.

Reply:
> Is this swagger spec file stored somewhere else? 
> Or because BDK is open source and groups API swagger is not?

In our internal GH org 

> Could you confirm it is copied here because of the securitySchemes stuff?

Nope, they haven't been modified

> Could we put it in the swagger-api-spec repo?

Good idea yes ! 👍